### PR TITLE
Add rocky9 support and update to using docker compose v2

### PR DESCRIPTION
**Pull request by nzlosh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Current community packages are hosted on https://packagecloud.io/StackStorm. For
 
 If you are adding support for a new distribution for which ``packagingbuild`` and ``packagingtest``
 images are not yet published to Docker Hub and you want to test the build pipeline locally, you
-need to update ``docker-compose.yml`` file to use locally built Docker images.
+need to update ``docker compose.yml`` file to use locally built Docker images.
```
Comment (mamercad) [inline README.md:109]: ``docker-compose.yml``

**Review by mamercad — CHANGES_REQUESTED** [review]
Small search and replace problem, other than that, excited that we're getting there for new OSs.

File: README.md (hunk 1 above)
Comment (nzlosh) [reply README.md:109]: Oh good spotting.  The entire sentence should be re-written.  docker-compose is v1 and is no longer functional with circle ci deprecating docker <1.20.  The docker compose plugin is required for v2 support, which available from the docker repository.

**Review by mamercad — APPROVED** [review]
🚢

**Pull request merged**